pom.xml

```diff
@@ -91,21 +80,14 @@
                     <suppressionsFileExpression>checkstyle.suppressions.file</suppressionsFileExpression>
                 </configuration>
             </plugin>
-            <!-- Begin fix for https://issues.jenkins-ci.org/browse/INFRA-588 -->
+            <!-- TODO: Remove once Javadoc is fixed -->
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-deploy-plugin</artifactId>
-                <version>2.6</version>
-                <dependencies>
-                    <dependency>
-                        <groupId>org.apache.maven.wagon</groupId>
-                        <artifactId>wagon-http</artifactId>
-                        <version>2.10</version>
-                        <type>jar</type>
-                    </dependency>
-                </dependencies>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <configuration>
+                    <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
We probably should add it to a property-triggered profile in ParentPOM. Hit it multiple times while working on JEP-200

Reply:
-1, just fix the silly `error`s already, it takes maybe a minute and you are done.